ngraph/frontend/frontend_manager/src/frontend_manager.cpp

```diff
@@ -96,14 +105,15 @@ FrontEndManager& FrontEndManager::operator=(FrontEndManager&&) = default;
 
 FrontEndManager::~FrontEndManager() = default;
 
-FrontEnd::Ptr FrontEndManager::load_by_framework(const std::string& framework, FrontEndCapFlags fec)
+FrontEnd::Ptr FrontEndManager::load_by_framework(const std::string& framework)
 {
-    return m_impl->loadByFramework(framework, fec);
+    return m_impl->loadByFramework(framework);
 }
 
-FrontEnd::Ptr FrontEndManager::load_by_model(const std::string& path, FrontEndCapFlags fec)
+FrontEnd::Ptr
+    FrontEndManager::load_by_variants(const std::vector<std::shared_ptr<Variant>>& variants)
```

Comment:
I think that `load_by_variants` is not clear name. Maybe can we use more clear name `arguments` or something like that?